examples/tls-authentication/nginx/README.md

```diff
@@ -0,0 +1,73 @@
+# TLS termination
```

Comment:
This title is wrong, and actually I think a better place to keep this example would be in a auth directory, so I'd imagine something like: 
```
auth
  oauth
  password
  client-certs
```

Wdyt? 
We could also instead create a tls directory like 
```
tls
  termination
  client-certs
  reencrypt ...
```